tools/autotag/templates/rocm_changes/5.0.0.md

```diff
@@ -301,38 +306,44 @@ The correct order is:
 
 Dispatch < Start < End < Complete
 
-Users cannot use ROCProfiler to measure the time spent on each kernel because of the incorrect timestamp with counter collection enabled.
+Users cannot use ROCProfiler to measure the time spent on each kernel because of the incorrect
+timestamp with counter collection enabled.
 
 ##### Recommended workaround
 
-Users are recommended to collect kernel execution timestamps without monitoring counters, as follows:
+Users are recommended to collect kernel execution timestamps without monitoring counters, as
+follows:
 
 1. ​Enable timing using the --timestamp on flag, and run the application.
 
-2. Rerun the application using the -i option with the input filename that contains the name of the counter(s) to monitor, and save this to a different output file using the -o flag.
+2. Rerun the application using the -i option with the input filename that contains the name of the
+  counter(s) to monitor, and save this to a different output file using the -o flag.
 
 3. Check the output result file from step 1.
 
-4. The order of timestamps correctly displays as:
-  DispatchNS < BeginNS < EndNS < CompleteNS
+4. The order of timestamps correctly displays as: DispatchNS < BeginNS < EndNS < CompleteNS
 
-5. Users can find the values of the collected counters in the output file generated in step 2.
+1. Users can find the values of the collected counters in the output file generated in step 2.
```

Comment:
We are not restarting the sequence of steps. Perhaps a note?

Reply:
This was VScode trying to be helpful and auto-updating my numbering. It should be 5.

Reply:
:) Been there, done that.